On Mon, Jun 23, 2008 at 04:20:16PM +0000, The Fungi wrote:
> On Sun, Jun 22, 2008 at 02:24:42PM +0200, Martin Michlmayr wrote:
> > I've built some kernel snapshots for mips and mipsel based on
> > 2.6.26-rc7.  Please test them on your favourite MIPS machine
> > and report success or failures to this list.
> 
> My Indy (CMN B006Y87, R4400 175MHz CPU) panics on the r4k-ip22
> package shortly after boot:

this might be due to the change in using a cache invalidate instead
of writeback/invalidate and the 128 byte linesize of the second level
cache on that R4k CPU modules. The patch below should fix that. 
I have no R4400 IP22 handy, so I'd appriciate if someone could test
the patch. If it works, I'll sent the patch to Ralf with proper S-O-B.

Thomas.

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index e5a7c5d..24c5dee 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1006,7 +1006,7 @@ config BOOT_ELF32
 config MIPS_L1_CACHE_SHIFT
 	int
 	default "4" if MACH_DECSTATION
-	default "7" if SGI_IP27 || SGI_IP28 || SNI_RM
+	default "7" if SGI_IP22 || SGI_IP27 || SGI_IP28 || SNI_RM
 	default "4" if PMC_MSP4200_EVAL
 	default "5"
 


-- 
Crap can work. Given enough thrust pigs will fly, but it's not necessary a
good idea.                                                [ RFC1925, 2.3 ]


-- 
To UNSUBSCRIBE, email to debian-mips-REQUEST@lists.debian.org
with a subject of "unsubscribe". Trouble? Contact listmaster@lists.debian.org
